composer.lock

```diff
@@ -4,21 +4,21 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "2a41918354598624d6717c6f006dcfff",
+    "content-hash": "babde470b9005d9f3d3b3f5ab88c3496",
```

Comment:
I am not sure composer.lock should be updated, it think modifying `type` and `suggest` fields should not trigger composer.lock update.